src/builders/scene/addEnvironment.ts

```diff
@@ -0,0 +1,29 @@
+import { Scene, WebGLRenderer } from "three"
+
+import { PMREMGenerator } from "three/examples/jsm/pmrem/PMREMGenerator"
+import { PMREMCubeUVPacker } from "three/examples/jsm/pmrem/PMREMCubeUVPacker"
+import { EquirectangularToCubeGenerator } from "three/examples/jsm/loaders/EquirectangularToCubeGenerator"
+import GameFieldAssets from "../../loaders/scenes/GameFieldAssets"
+
+export const addEnvironment = (scene: Scene, renderer: WebGLRenderer) => {
+  const { environment } = GameFieldAssets.getAssets()
+
+  const cubeGenerator = new EquirectangularToCubeGenerator( environment, { resolution: 2048 } );
```

Comment:
`EquirectangularToCubeGenerator` was removed in three.js 110, use `WebGLRenderTargetCube.fromEquirectangularTexture()`

```ts
const cube = new WebGLRenderTargetCube(2048, 2048).fromEquirectangularTexture(renderer, environment)
scene.background = cube

const pmremGenerator = new PMREMGenerator(cube.texture)
```

Reply:
You are right, I noticed the change right after.